source/plugin/workspace/idea.rst

```diff
@@ -72,13 +81,42 @@ Importing your project
 If you've already started with your project and want to import it again at a later point you need to import it instead
 of re-creating it inside your IDE:
 
-* Click ``File > Open``
-* Navigate to the project's ``pom.xml`` file
-* Click Ok
+* Click ``File > Open`` or ``Import Project``.
+* Navigate to the project's ``pom.xml`` file and select it.
+* Make sure the settings are as you desire and click ``Ok``.
+
+Minecraft Dev Plugin
+====================
+
+The Minecraft Development plugin for IntelliJ is a great plugin by a community member which makes plugin project
+creation much easier while also providing some neat and useful features for development.
+
+Creating your project
+~~~~~~~~~~~~~~~~~~~~~
+* Open **IntelliJ IDEA**.
+* Click ``Create New Project``.
+* Select ``Gradle`` in the popup.
```

Comment:
Should be "Minecraft Plugin".

Reply:
Ah yeah, forgot to edit that after copying.